package.json

```diff
@@ -57,6 +60,6 @@
     "protractor": "~5.1.2",
     "ts-node": "~3.2.0",
     "tslint": "~5.9.1",
-    "typescript": "^2.5.3"
+    "typescript": "^2.7.2"
```

Comment:
I think we need to remove the caret here to ensure that it matches the CLI requirements:
````
ERROR in The Angular Compiler requires TypeScript >=2.7.2 and <2.8.0 but 2.9.2 was found instead.
````